src/encoded/static/components/objectutils.js

```diff
@@ -862,7 +862,7 @@ export function computeAssemblyAnnotationValue(assembly, annotation) {
     // First level of sorting: most recent assemblies are ordered first (represented by numerical component of assembly)
     // Second level of sorting: assemblies without '-minimal' are sorted before assemblies with '-minimal' at the end (represented by tenths place value which is 5 if there is no '-minimal')
     // Third level of sorting: Annotations within an assembly are ordered with most recent first, with more recent annotations having a higher annotation number (with the exception of "ENSEMBL V65") (represented by the annotation number divided by 10,000, or, the three decimal places after the tenths place)
-    let assemblyNumber = +assembly.match(/[0-9]+/g)[0];
+    let assemblyNumber = Number.isNaN(Number(assembly)) ? 0 : +assembly.match(/[0-9]+/g)[0];
```

Comment:
When would Number(assembly) ever have any value other than NaN? I think `assemblyNumber` always gets the value 0.

The cart uses this function for sorting assemblies. With dev I get:

<img width="300" alt="dev" src="https://user-images.githubusercontent.com/1181324/123702180-acce9800-d817-11eb-8095-f6965d7c2168.png">

…and with this branch I get:

<img width="300" alt="branch" src="https://user-images.githubusercontent.com/1181324/123702207-b35d0f80-d817-11eb-91b6-71f3ecdcd3cc.png">

…which seems wrong.